images/ansible-operator/Dockerfile

```diff
@@ -0,0 +1,46 @@
+FROM registry.access.redhat.com/ubi8/ubi:latest
+
+RUN mkdir -p /etc/ansible \
+  && echo "localhost ansible_connection=local" > /etc/ansible/hosts \
+  && echo '[defaults]' > /etc/ansible/ansible.cfg \
+  && echo 'roles_path = /opt/ansible/roles' >> /etc/ansible/ansible.cfg \
+  && echo 'library = /usr/share/ansible/openshift' >> /etc/ansible/ansible.cfg
+
+ENV HOME=/opt/ansible \
+    USER_NAME=ansible \
+    USER_UID=1001
+
+# Install python dependencies
+# Ensure fresh metadata rather than cached metadata in the base by running
+# yum clean all && rm -rf /var/yum/cache/* first
+RUN yum clean all && rm -rf /var/cache/yum/* \
+  && yum -y update \
+  && yum install -y libffi-devel openssl-devel python36-devel gcc python3-pip python3-setuptools \
+  && pip3 install --no-cache-dir \
+    ipaddress \
+    ansible-runner==1.3.4 \
+    ansible-runner-http==1.0.0 \
+    openshift~=0.10.0 \
+    ansible~=2.9 \
```

Comment:
I know we're just merging commit wholesale but this needs to be pinned to 2.9.15 (see https://github.com/operator-framework/operator-sdk/pull/4321). We can either wait for v1.2.1 or v1.3. Thoughts @jmrodri?

/ping @asmacdo

Reply:
Those aren't being used downstream. I probably need to compare the files on release/ansible

Reply:
@estroz I've compared the 2. The downstream has its own dockerfiles https://github.com/openshift/ocp-release-operator-sdk/tree/master/release I usually just compare them to see if there's anything significant that needs to be brought over. The versions of ansible etc, are all controlled by the packages in the downstream repo which are up to date.

Reply:
Cool, resolved.